recipe/build.sh

```diff
@@ -1,9 +1,39 @@
  #!/usr/bin/env bash
 set -ex
 
+# Stopgap while https://github.com/conda-forge/libiconv-feedstock/pull/46
+# it will have a similar file generated
+libiconv_pc_created=false
+if [[ "${libiconv}" != "" ]] && [[ -f ${PREFIX}/lib/pkgconfig/libiconv.pc ]]; then
+    libiconv_pc_created=true
+    echo >${PREFIX}/lib/pkgconfig/libiconv.pc <<EOF
+prefix=${PREFIX}
+exec_prefix=\${prefix}
+libdir=\${exec_prefix}/lib
+includedir=\${prefix}/include
+Name: libiconv
```

Comment:
```suggestion
Name: iconv
```

Reply:
ah thanks...

Reply:
should the name of the file match? i now named it (after you looked at things) as `iconv.pc` instead of `libiconv.pc`.

Reply:
Indeed, looks like it should also be named `iconv.pc`.